css/_uswds-theme-custom-styles.scss

```diff
@@ -1668,7 +1672,7 @@ code[class*="language-plaintext"] {
   margin-left: units(3);
 }
 
-.link_group-download {
+.link__group-download {
```

Comment:
Class was updated in CSS, but not in markup. I still see templates that use `link_group-download`.